lib/web3.js

```diff
@@ -78,56 +79,60 @@ var setupProperties = function (obj, properties) {
 };
 
 /// setups web3 object, and it's in-browser executed methods
-var web3 = {};
-web3.providers = {};
-web3.currentProvider = null;
-web3.version = {};
-web3.version.api = version.version;
-web3.eth = {};
+function Web3(provider) {
```

Comment:
That's a constructor for `web3` object. The object stores links to `requestManager` and namespaces like `eth` (`eth1` in this case).